cypress/e2e/gui/detalheCliente.cy.js

```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
+describe('Detalhes do cliente', () => {
+    beforeEach(() => {
+        cy.visit('/')
+    })
```

Comment:
```suggestion
    })

```

Deixar uma linha em branco entre o `beforeEach` e os blocos `it` (ou `context`) é uma convenção que ajuda na legibilidade do código e separação das diferentes partes da suíte.

Reply:
Obrigada pela dica!